docs/server/project_setup.rst

```diff
@@ -116,7 +116,25 @@ unless you need to further customize your project.
 
 
 Once you are done click on the **Save** button below the form to create the
-project. Creating the project doesn't actually import all the translations to
+project.
+
+Please not that **just creating the project is not enough**, now it is
+necessary to set the the filesystem configuration. So now click on the
+``Filesystems`` link below the project edit form and set the following:
+
+.. image:: ../_static/pootle_fs_link.png
+
+
+* **Filesystem backend** to ``localfs``
+* **Backend URL or path** to the value of
+  :setting:`POOTLE_TRANSLATION_DIRECTORY` + project code, e.g.
+  :file:`/path/to/pootle/translations/my-project` in our example
+* **Translation mapping presets** to ``GNU-style``
+
+And then click on the **Update filesystem configuration** button to save the
+Pootle FS configuration for the project.
+
+Creating the project doesn't actually import all the translations to
 Pootle. To do that you need to run :djadmin:`update_stores` on the command line
```

Comment:
Are we going to revisit the `update_stores` part later?

Reply:
Given that these commands are working with Pootle FS I was wondering if we actually want to drop them now that everything *works* with Pootle FS or if they would stay around a bit more.

So in short this stays as it is until we agree when to deprecate the commands formally.